static/fluent/sr/main.ftl

```diff
@@ -1,9 +1,21 @@
+## The following terms must be treated as brand, and kept in English.
+##
+## They cannot be:
+## - Transliterated.
+## - Translated.
+##
+## Declension should be avoided where possible.
+##
+## Reference: https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#brands-copyright-and-trademark
+
+-webthings-gateway-brand = { -webthings-gateway-brand }
```

Comment:
Should this be `-webthings-gateway-brand = WebThings Gateway` ?

Reply:
Yes, great catch! Fixed.